tests/avify4mtest.c

```diff
@@ -161,11 +161,16 @@ static avifBool encodeDecodeY4m(uint32_t width,
 int main(int argc, char * argv[])
 {
     if (argc != 2 || !strlen(argv[1])) {
-        fprintf(stderr, "Missing temporary folder path argument\n");
+        fprintf(stderr, "Missing temporary directory environment variable argument\n");
```

Comment:
"Missing temporary directory path environment variable name argument"?

Reply:
Done. The error message is now precise but a little hard to parse.